### PR TITLE
fix: make link_audio_delay_list and sound_program_list optional

### PR DESCRIPTION
**Pull request by josser**

### CODE REVIEW

**Comment by josser** [comment]
Hi! I have R-N602 receiver and it doesn't have sound_program_list / link_audio_delay_list options, so I made them optional

**Comment by cgierke** [comment]
cool, thanks a lot!

File: src/YamahaAPI.ts [hunk 1]
```diff
@@ -67,12 +67,12 @@ interface Response {
 export interface StatusResponse {
     response_code: number;
     power: string;
-    //sleep: number;
+    sleep: number;
```
Comment (cgierke) [inline src/YamahaAPI.ts]: probably that was not intended to be in this PR?
```suggestion
    //sleep: number;
```

File: src/YamahaAPI.ts [hunk 2]
```diff
@@ -67,12 +67,12 @@ interface Response {
...
     volume: number;
     mute: boolean;
     max_volume: number;
     input: string;
-    input_text: string;
+    input_text?: string;
```
Comment (cgierke) [inline src/YamahaAPI.ts]: probably that was not intended to be in this PR?
```suggestion
    input_text: string;
```

File: src/YamahaAPI.ts (hunk 1 above)
Comment (josser) [reply src/YamahaAPI.ts]: Just though it can be maybe useful if I want to implement some additional features )

File: src/YamahaAPI.ts (hunk 2 above)
Comment (josser) [reply src/YamahaAPI.ts]: This is what I see when I'm doing this call for R-N602, so I decided to left it

**Pull request merged**